chart-infra/templates/kube-cleanup-operator-deployment.yaml

```diff
@@ -33,6 +33,7 @@ spec:
         # new workers in its place
         - -keep-successful=0
         - -keep-failures=0
+        - -dry-run
```

Comment:
what does this do and how does that help us ?

Reply:
Sorry for the noise. Yesterday I was trying to get to see the logs of completed jobs, in order to understand my issues and the issues that @JuanluOnieva-biomage was having with the worker, so I thought about unplugging the cleanup. Somehow I was unable to have the ci pipeline run on this and have it deployed.